testpmd/hooks/pre-run.yml

```diff
@@ -20,6 +9,50 @@
       - name: intel-numa0-net2
         count: 2
     mac_workaround_enable: false
+    cnf_namespace: "example-cnf"
+
+- name: Get example-cnf component details from job.components
+  set_fact:
+    operator_version: "{{ item['version'] }}"
+    example_cnf_index_image: "{{ item['data']['url'] }}:{{ item['version'] }}"
+  with_items: "{{ hostvars.localhost.job_info.job.components }}"
+  when: item["type"] == "nfv-example-cnf-index"
+
+- name: Check change presence
+  block:
+    - name: Check if example-cnf is part of the change
+      stat:
+        path: "{{ dci_change_dir }}/example-cnf"
```

Comment:
IIUC, if there is a dci change on example-cnf repo, then the example-cnf  folder is present because of the test-runner script.

Reply:
yes exactly